sspl_test/rabbitmq/rabbitmq_ingress_processor_tests.py

```diff
@@ -199,30 +200,30 @@ def _process_msg(self, ch, method, properties, body):
     def _configure_exchange(self):
         """Configure the RabbitMQ exchange with defaults available"""
         try:
-            self._virtual_host = self._conf_reader._get_value_with_default(
-                self.RABBITMQPROCESSORTEST, self.VIRT_HOST, "SSPL"
+            self._virtual_host = Conf.get("SSPL-Test", 
+                            f"{self.RABBITMQPROCESSORTEST}>{self.VIRT_HOST}","SSPL"

```

Comment:
avoid using f"" in new code atleast, as per the new guidlines. Such string formatting is likely to create problems during localization efforts later as per @ujjwalpl 
Address everywhere, used in this patch

Reply:
In this PR, all changes made for f string are to generate string to get config from confstor.
No exception handling or logging string changed
Do I need to remove f string still?

Reply:
You may ignore this comment, as i mentioned in Team's chat couple of days back. We need to avoid using f" for any string which may be visible to customers through GUI or CLI, which would need localization.